docs/README.md

```diff
@@ -1,39 +1,5 @@
-# Documentation Index
+# Documentation
 
-## Getting started
+## User Documentation
 
-* [Frequently Asked Questions](frequently_asked_questions.md)
-* Quickstart guides
-  * [AWS](quickstart-aws.md)
-  * [Azure](quickstart-azure.md)
-  * [DigitalOcean](quickstart-digitalocean.md)
-  * [Google Compute Engine (GCE)](quickstart-gce.md)
-  * [Hetzner](quickstart-hetzner.md)
-  * [OpenStack](quickstart-openstack.md)
-  * [Packet](quickstart-packet.md)
-  * [VMware vSphere](quickstart-vsphere.md)
-
-## Using KubeOne
-
-* [SSH requirements](ssh.md)
-* [Upgrading Kubernetes Clusters Using KubeOne](upgrading_cluster.md)
-* [Using Addons](addons.md)
-* [Proxy support](proxy.md)
-* [Manual Cluster Repair](manual-cluster-repair.md)
-* [Environment variables used by KubeOne](environment_variables.md)
-* [Terraform LoadBalancers in Examples](example-loadbalancer.md)
-* [Migrating to the KubeOneCluster API](api_migration.md)
-
-## Development
-
-* [Adding support for a provider](adding_provider_support.md)
-* [Project Structure](project_structure.md)
-* [Backwards Compatibility Policy](backwards_compatibility_policy.md)
-* [Release Process](release_process.md)
-
-## [Proposals](./proposals)
-
-* [Cluster Upgrades (implemented)](./proposals/20190211-upgrades.md)
-* [KubeOneCluster API (implemented)](./proposals/20190409-kubeonecluster-api.md)
-* [KubeOne Addons (implemented)](./proposals/20200205-addons.md)
-* [KubeOne Reconciliation process (draft)](./proposals/20200224-apply.md)
```

Comment:
This is still relevant, so I'd leave it in the index, such as `Development Documentation`.